pkg/cmd/server/origin/controller/autoscaling.go

```diff
@@ -44,14 +38,30 @@ func (c *HorizontalPodAutoscalerControllerConfig) RunController(originCtx Contro
 		"heapster",
 		"",
 	)
-	replicaCalc := hpacontroller.NewReplicaCalculator(metricsClient, hpaClient.Core())
+	replicaCalc := hpacontroller.NewReplicaCalculator(
+		metricsClient,
+		hpaClient.CoreV1(),
+		0.1, // this is the default
```

Comment:
we'll eventually want to plumb this back to the controller arg, but hopefully we shouldn't have any of this for too long